cli/src/main/java/de/jplag/cli/options/CliOptions.java

```diff
@@ -107,6 +108,9 @@ public static class Advanced {
 
         @Option(names = "--log-level", description = "Set the log level for the cli.")
         public Level logLevel = Level.INFO;
+
+        @Option(names = "--submission-charset", description = "Specifies a charset to be used for submissions. This disables the automatic charset detection for submissions.", completionCandidates = CharsetCandidates.class, converter = CharsetConverter.class)
```

Comment:
```suggestion
        @Option(names = "--encoding", description = "Specifies the charset of the submissions. This disables the automatic charset detection", completionCandidates = CharsetCandidates.class, converter = CharsetConverter.class)
```

I would be in favor of a shorter CLI parameter, e.g. `encoding` or similar.